aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -928,6 +935,31 @@ SparseTensor& mul_out_sparse_cpu(const Tensor& t_, const Tensor& src_, Tensor& r
     return _mul_dense_sparse_out(t_, src_, r);
   }
 
+  // case mul(sparse, sparse) with a 0-dim input.
+  const auto get_vals_from_scalar = [](const SparseTensor& s) -> Tensor {
+    const auto vals = s._values().squeeze(0);
+    if (vals.dim()) {
```

Comment:
when would this happen? (just point me to a test case if you've got one)

Reply:
Yes it happens in test_basic_ops. You can have a sparse tensor of shape () with values of dim 1 and shape [0].